src/components/Button.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import PropType from 'prop-types';
 
-const Button = ({ buttonName }) => (
-  <button type="button">{buttonName}</button>
+const Button = ({ buttonName, color = 'orange' }) => (
```

Comment:
- [ ] You shouldn't set the value of `color` directly as such. Use `defaultProps` instead. You can read about it [here](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values).

Reply:
changed

Reply:
Refactored